spec/unit/govuk_index/page_traffic_loader_spec.rb

```diff
@@ -20,10 +20,11 @@
     line2 = [{ "val" => "c" }, { "data" => 1 }, { "val" => "d" }, { "data" => 1 }]
     line3 = [{ "val" => "e" }, { "data" => 1 }]
 
-    expect(GovukIndex::PageTrafficWorker).to receive(:perform_async).with(line1, 'new_index_name')
-    expect(GovukIndex::PageTrafficWorker).to receive(:perform_async).with(line2, 'new_index_name')
-    expect(GovukIndex::PageTrafficWorker).to receive(:perform_async).with(line3, 'new_index_name')
-
+    Clusters.active.each { |cluster|
```

Comment:
I think `do`/`end` is a bit clearer for a multi-line thing like this.  Also, you have angered the linter